python/pyspark/sql/tests/test_arrow.py

```diff
@@ -383,6 +383,18 @@ def test_timestamp_dst(self):
         assert_frame_equal(pdf, df_from_python.toPandas())
         assert_frame_equal(pdf, df_from_pandas.toPandas())
 
+    def test_timestamp_nat(self):
+        dt = [pd.NaT, pd.Timestamp('2019-06-11'), None] * 100
+        pdf = pd.DataFrame({'time': dt})
+
+        with self.sql_conf({'spark.sql.execution.arrow.pyspark.enabled': "false"}):
```

Comment:
not a big deal but I think we can do this with a for loop.

Reply:
Good point refactored to forloop